index.js

```diff
@@ -1,3 +1,131 @@
+ javascript-javascript1-week3
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+const index = names.indexOf(nameToRemove);
+if (index !== -1) {
+  names.splice(index, 1);
+}
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function calculateTravelTime(travelInformation) {
+  const speed = travelInformation.speed;
+  const destination = travelInformation.destinationDistance;
+  let time = destination / speed;
+  const hours = Math.floor(time);
+  const minutes = Math.round((time - hours) * 60);
+  return `${hours} hours and ${minutes} minutes`;
+}
+
+const travelTime = calculateTravelTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
+//Series duration of my life
+const seriesDurations = [
```

Comment:
Well done with this exercise. You've done super well. Only thing I could recommend you doing is setting `Prettier` to run on save, or maybe even setting Auto Save so you don't need to remember to format the code. It will make the code a bit more readable and will potentially save you time from evening out lines and adding or removing spacing here and there. Here is a guide instructing how to set this up - https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code